packages/core/src/App/Components/Elements/NotificationMessage/notification.jsx

```diff
@@ -61,6 +64,9 @@ const Notification = ({ data, removeNotificationMessage }) => {
                                 timeout={data.timeout}
                                 action={data.action.onClick}
                                 render={data.timeoutMessage}
+                                should_store_in_session={true}
+                                session_id={(session_id += 1)}
```

Comment:
```suggestion
                                session_id={data.key}
```
You can use `data.key`, it's a unique key for notification. 
![image](https://user-images.githubusercontent.com/56330681/130599374-4475bc06-2cc0-49e9-81c1-1dd766395f1a.png)

Reply:
Ah ofcourse, thanks. fixed this now.